src/components/molecules/Visualizer/Widget/Storytelling/index.tsx

```diff
@@ -130,19 +149,19 @@ const Wrapper = styled.div`
   }
 `;
 
-const ArrowButton = styled.button`
-  background-color: ${props => props.theme.main.paleBg};
+const ArrowButton = styled.button<{ publishedTheme: PublishTheme }>`
+  color: ${({ publishedTheme }) => publishedTheme.mask};
```

Comment:
Is passing publishedTheme into all the styled components necessary? For example, this ArrowButton had `color: inherit` but you replaced it with the `publishedTheme.mainIcon`.
If you can check if everything inherits from the Wrapper's color attribute that would clean up this code a lot.

Reply:
Unfortunately, every component is using different set of styles...
I have checked again and there is no way to inherit because child may using `mainIcon` and parent is using `mainText` etc...